projects/build-openssl.bat

```diff
@@ -44,7 +44,11 @@ rem ***************************************************************************
     set OS_PLATFORM=x86
   )
   if defined PROGRAMFILES(x86) (
-    set "PF=%PROGRAMFILES(x86)%"
+    if /i "%~1" == "vc14.3" (
+      set "PF=%PROGRAMFILES%"
+    ) else (
+      set "PF=%PROGRAMFILES(x86)%"
+    )
```

Comment:
why is this necessary, and same question for same change in build-wolfssl

Reply:
Visual Studio used to be x86 only up to VS17